examples/restaurant_visits/run_without_frameworks_utility_analysis.py

```diff
@@ -31,27 +32,21 @@
 flags.DEFINE_string('output_file', None, 'Output file')
 flags.DEFINE_boolean('public_partitions', False,
                      'Whether public partitions are used')
+flags.DEFINE_boolean(
+    'per_partitions_metrics', False,
+    'Whether per partition or aggregate utility analysis is computed')
```

Comment:
Does this refer to different types of utility analysis: per partition and aggregated? Just for my understanding, what would be an example of each?

Reply:
Yes.
Cross-partition aggregated example:
```
(PartitionSelectionMetrics(num_partitions=7, dropped_partitions_expected=0.0, dropped_partitions_variance=0.0), 
 AggregateErrorMetrics(abs_error_expected=-619.9999999999998, abs_error_variance=131.3383283342634, abs_error_quantiles=[-605.3363229857413, -619.9999999999998, -634.6636770142583, -646.618370079588], rel_error_expected=-0.7623124338624336, rel_error_variance=0.000213443176382021, rel_error_quantiles=[-0.7437896396105713, -0.7623124338624336, -0.7808352281142958, -0.7959361008202313]))
```

Per partition:
```
(1, (1.0, CountUtilityAnalysisMetrics(count=600, per_partition_error=-300, expected_cross_partition_error=-162.80000000000007, std_cross_partition_error=7.645187302407234, std_noise=8.3515625, noise_kind=<NoiseKind.GAUSSIAN: 'gaussian'>)))
(2, (1.0, CountUtilityAnalysisMetrics(count=600, per_partition_error=-300, expected_cross_partition_error=-161.3666666666666, std_cross_partition_error=7.506737714287814, std_noise=8.3515625, noise_kind=<NoiseKind.GAUSSIAN: 'gaussian'>)))
... # other partitions
```
i.e. per-partition contains metrics per each partition

Reply:
I see, thanks for clarifying!